scheduler/src/cook/mesos/util.clj

```diff
@@ -337,20 +345,45 @@
 (def ^:const job-states #{"running" "waiting" "completed"})
 (def ^:const instance-states #{"success" "failed"})
 
-;; get-completed-jobs-by-user is a bit opaque because it is
```

Comment:
I think this comment is probably worth keeping. I believe it was added because reviewers when it was added requested an explanation of why the code worked. @dposada what do you think?

Reply:
I agree, this comment seems very helpful to me

Reply:
Resurrected